src/components/hw-design/hw-input.tsx

```diff
@@ -0,0 +1,33 @@
+import { InputHTMLAttributes, ReactNode } from "react";
+import { Input } from "../ui/input";
+
+function HWInput({
+  title,
+  placeholder,
+  subtitle,
+  className = "",
+  variant,
+  children,
+  ...props
+}: {
+  title: string;
+  placeholder: string;
+  subtitle: string;
+  className?: string;
+  children?: ReactNode;
+  variant: "primary" | "destructive" | "outline" | "subtle";
+} & InputHTMLAttributes<HTMLInputElement>) {
+  return (
+    <div className="flex w-full flex-col gap-2">
+      <p className="pl-1 font-medium">{title}</p>
+      <Input
+        className="rounded-xl bg-primary-50 px-5 py-2 text-black outline outline-2 outline-primary-300 hover:bg-primary-200"
+        placeholder={placeholder}
+        {...props}
+      />
+      <p className="pl-1 font-light text-gray-700">{subtitle}</p>
```

Comment:
ok, my other problem with this component is the titles, I think that's not a good abstraction. if we want to customize the label for example.

I know this is dumb, but semantically using paragraph elements for the labels is not great for accessibility either since they're not linked to the actual input element.

Reply:
Also, when we add more shadcn components, by default it uses any dependent components from the ui folder.

Reply:
The other issue is that other shadcn components that we will add in the future won't use this one, so they won't have the default styling that we want.